### PR TITLE
Fix typos in README.md and tighten up language use

### PR DESCRIPTION
**Pull request by yesmar**
Here is a small contribution to clean up some of the language in the `README.md` file. There were some typos, some awkward phrasing, etc. I made corrections in the How To Play, Controller support, and F.A.Q. sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,20 +267,20 @@ variable (see
...
+No, you'll need access to the data from the original game. If you don't have an original CD then you can [buy Diablo from GoG.com](https://www.gog.com/game/diablo). Alternately you can use `spawn.mpq` from the [http://ftp.blizzard.com/pub/demos/diablosw.exe](shareware) version and compile the with the SPAWN flag defined.
 > What game changes does DevilutionX provide
 
-DevilutionX's main focuse is to making the game work on multiple platforms and making the engine mod frindly. As such there are not change to the gameplay, but we will be making some enhancments to the engine it self. To give some examples there is now upscaling, unlocked fps and controller support and multiplayer via TCP.
+DevilutionX's main focus is to make the game work on multiple platforms. An additional goal is to make the engine mod friendly. As such, there are not many changes to gameplay, but we will be making some enhancments to the engine itself. For example, the engine now has upscaling, unlocked fps, controller support, and multiplayer via TCP.
```
Comment (AJenbo) [inline README.md]:
```suggestion
DevilutionX's main focus is to make the game work on multiple platforms. An additional goal is to make the engine mod friendly. As such, there are no changes to gameplay, but we will be making some enhancments to the engine itself. For example, the engine now has upscaling, unlocked fps, controller support, and multiplayer via TCP.
```

**Comment by AJenbo** [comment]
Thank you for this, writing copy is not my favorite part of this project :)

Would you be willing to copy the relevant corrections to the readme files in the various packaging folder? https://github.com/diasurgical/devilutionX/tree/master/Packaging

**Comment by yesmar** [comment]
OK, I think I got all of the necessary changes in place. Let me know if I missed anything.

**Comment by AJenbo** [comment]
Thank you very much for taking the time to do this :+1:

**Comment by yesmar** [comment]
I'll be back to do more.

**Pull request merged**